manifests/globals.pp

```diff
@@ -94,7 +94,13 @@
     'FreeBSD' => '93',
     'OpenBSD' => '9.3',
     'Suse' => $::operatingsystem ? {
-      'SLES' => '91',
+      'SLES' => $::operatingsystemrelease ? {
+        '11.3' => '91',
```

Comment:
This line can be removed now? Also, how is this working in future versions of SLES? Will 12.1 have 91 again, or does that need another patch?

Using [versioncmp](https://docs.puppetlabs.com/references/latest/function.html#versioncmp) would make that much nicer.